radix-engine/src/system/kernel_modules/kernel_debug/module.rs

```diff
@@ -109,6 +109,7 @@ impl KernelModule for KernelDebugModule {
         offset: &SubstateOffset,
         flags: &LockFlags,
     ) -> Result<(), RuntimeError> {
+        /*
```

Comment:
Any reason we disable it? I've been relying on them for debugging.

Reply:
Yes, I've been disabling Substate Lock/Read/Write/Unlock logs as it's taken up too much log space. But since you're still using it I'll reenable these back and we can have discussion on best strategy here.